config/optional/amp.theme.yml

```diff
@@ -0,0 +1 @@
+amptheme: thunder_amp
```

Comment:
This has no dependency at all. Wouldn't that be configured, even if I install something else?